docs/api.html

```diff
@@ -48,14 +49,14 @@
 	The Consumer API allows applications to read streams of data from topics in the Kafka cluster.
 	<p>
 	Examples showing how to use the consumer are given in the
-	<a href="/{{version}}/javadoc/index.html?org/apache/kafka/clients/consumer/KafkaConsumer.html" title="Kafka 0.10.2 Javadoc">javadocs</a>.
+	<a href="/{{version}}/javadoc/index.html?org/apache/kafka/clients/consumer/KafkaConsumer.html" title="Kafka {{version}} Javadoc">javadocs</a>.
```

Comment:
I just generated docs from this locally and it seems fine (which makes sense -- it's just a template so it should work anywhere). However, note that this doesn't quite match the original -- it now says "Kafka 0102 Javadoc".